bower_components/angular/.bower.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "angular",
+  "version": "1.3.14",
+  "main": "./angular.js",
+  "ignore": [],
+  "dependencies": {},
+  "homepage": "https://github.com/angular/bower-angular",
+  "_release": "1.3.14",
+  "_resolution": {
+    "type": "version",
+    "tag": "v1.3.14",
+    "commit": "bdd91e1331b48ae89adfb8444998e290b774cc6d"
+  },
+  "_source": "git://github.com/angular/bower-angular.git",
+  "_target": "~1.3.14",
+  "_originalSource": "angular",
+  "_direct": true
+}
```

Comment:
Not sure how you ended up with a `.bower.json` file _in addition_ to the `bower.json` file for the project...